addons/account/exceptions.py

```diff
@@ -0,0 +1,30 @@
+from odoo.exceptions import ValidationError
+from odoo.tools.misc import format_date
+from odoo import fields, _
+
+
+class ImbalanceJournalEntryError(ValidationError):
+    """Specialized ValidationError raised by `_check_balanced` method."""
+
+    def __init__(self, imbalance_amounts):
```

Comment:
@smetl :)

Reply:
I don't understand why you need to get both amounts and moves. You can recompute them easily

Reply:
Just not to compute it twice and avoid duplicated code

Reply:
`sum(move.line_ids.mapped('debit')) / sum(move.line_ids.mapped('credit'))` is not much